.changeset/kind-jobs-camp.md

```diff
@@ -0,0 +1,13 @@
+---
+"@salt-ds/lab": patch
+---
+
+Changes to `CircularProgress` and `LinearProgress`:
+
+- Removed size prop
+- Added max prop for custom max value
+- Removed variant props
+
+Changes to `StatusIndicator`:
```

Comment:
This is misleading. StatusIndicator is a separate component and the actual change is to Content Status

Reply:
It was also an internal only component so actually had no user impact